src/GraphQLParser/Visitors/SDLPrinter.cs

```diff
@@ -1138,6 +1138,12 @@ private static bool TryPeekParent(TContext context, [NotNullWhen(true)] out ASTN
         return true;
     }
 
+    private static readonly string[] _hex32 = new[]
+    {
+        "00", "01", "02", "03", "04", "05", "06", "07", "08", "09", "0A", "0B", "0C", "0D", "0E", "0F",
+        "10", "11", "12", "13", "14", "15", "16", "17", "18", "19", "1A", "1B", "1C", "1D", "1E", "1F"
```

Comment:
If these are the equivalent of individual constants, shouldn't we test for all conditions?

Reply:
done